walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -2,26 +2,30 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Stanley 🐈</title>
+    <title>Jonathan Aner's Portfolio</title>
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
     <div id="content">
-      <h1>Stanley 🐈</h1>
-      <p>This is Stanley.</p>
-      <img src="images/stanley-1.jpg"/>
+      <h1>Jonathan Aner's Portfolio</h1>
+      <p>Welcome to my Portfolio</p>
 
-      <p>She is very ferocious!</p>
-      <img src="images/stanley-2.jpg"/>
-
-      <p>But only sometimes.</p>
-      <img src="images/stanley-3.jpg"/>
-
-      <p>Mostly she likes to eat food and sleep. Sometimes at the same time.</p>
-      <img src="images/stanley-6.jpg"/>
-
-      <p>Click <a href="images.html">here</a> to see more pictures of Stanley!</p>
-      <p>Click <a href="random.html">here</a> to see a random picture of Stanley!</p>
-    </div>
+      <p><b>What makes Me Me?</b> </p>
```

Comment:
You can use <h2> in this file as well (and essentially anywhere where you have subtitles)